NagorniyAleksandr/SQLTask/src/main/java/university/util/Validator.java

```diff
@@ -32,7 +32,7 @@ public static void validateStudent(Student student) throws InvalidValueException
             throw new InvalidValueException("Student is null");
         }
 
-        if (!PATTERN_FOR_PEOPLE_NAME.matcher(student.getName()).matches()) {
+        if (student.getName() == null || !PATTERN_FOR_PEOPLE_NAME.matcher(student.getName()).matches()) {
```

Comment:
and just call validator.validate(student)
Either get a confirmation or catch an exception